### DISCUSSION TITLE
What will it happen if mounted with a new redis？

### DISCUSSION THREAD

**Discussion by wyujie** [juicedata/juicefs#715] [General]
Hi, juice team,

Assuming my server is down and the meta data of Juice stored in redis are lost. What will it happen if I mount Juicefs using new redis?

Thanks.

**Comment by davies** [comment]
You can not mount with new redis before format it. Once you format a new redis, you will have a empty file system.

**Reply by wyujie** [reply]
Is it possible to use Juicefs to backup the redis data when Juicefs is running? Will it raise interdependent issue?

**Reply by SandyXSD** [reply]
JuiceFS doesn't back up Redis data, instead Redis itself has several strategies to keep data persistent, please check: [Redis best practice](https://github.com/juicedata/juicefs/blob/main/docs/en/redis_best_practices.md#data-durability).
To achieve higher reliability, you can also try other metadata engines, see [metadata engines](https://github.com/juicedata/juicefs/blob/main/docs/en/databases_for_metadata.md) and their [benchmarks](https://github.com/juicedata/juicefs/blob/main/docs/en/metadata_engines_benchmark.md).

**Reply by wyujie** [reply]
Get it, thanks for your kind reply.
